src/i18n/pl.json

```diff
@@ -264,5 +264,7 @@
   "year" : "Rok",
   "years" : "Lata",
   "zoom_in" : "Powiększ",
-  "zoom_out" : "Pomniejsz"
+  "zoom_out" : "Pomniejsz",
+  "dedup_results" : "Deduplikuj wyniki",
+  "duplicate" : "duplikować"
```

Comment:
shall it be it a verb or noun? above is a verb

Reply:
Thanks Mateusz!